src/main/java/io/codelex/arrays/exercises/Exercise8.java

```diff
@@ -0,0 +1,76 @@
+package io.codelex.arrays.exercises;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Exercise8 {
+    public static void main(String[] args) {
+        hangmanGame();
+    }
+
+    private static void hangmanGame() {
+        String[] listOfWords = {"airplane", "car", "computer", "window", "apple", "bicycle", "programming", "software", "mouse", "keyboard"};
+
+        Random rand = new Random();
+        int randomWordIndex = rand.nextInt(listOfWords.length);
+        String chosenRandomWord = listOfWords[randomWordIndex];
+        char[] wordArrOfChars = chosenRandomWord.toCharArray();
+        int limitedTries = 3;
+
+        StringBuilder word = new StringBuilder();
+        StringBuilder missesChar = new StringBuilder();
+        StringBuilder guessChar = new StringBuilder();
+
+        word.append("_".repeat(chosenRandomWord.length()));
+
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("-=-=-=-=-=-=-=-=-=-=-=-=-=-");
+        System.out.print("Word:   " + word + "\n");
+        System.out.print("Misses: " + missesChar + "\n");
+        System.out.print("Guess:  " + guessChar + "\n");
+        System.out.println("-=-=-=-=-=-=-=-=-=-=-=-=-=-");
+
+        while (!word.toString().equals(chosenRandomWord)) {
+            System.out.print("Please enter your guess character: ");
+            char guess = in.next().toLowerCase().charAt(0);
+            if (new String(wordArrOfChars).contains("" + guess)) {
+                for (int i = 0; i < wordArrOfChars.length; i++) {
+                    if (wordArrOfChars[i] == guess) {
+                        word.setCharAt(i, wordArrOfChars[i]);
+                    }
+                }
+                guessChar.append(guess);
+            } else {
+                missesChar.append(guess);
+                limitedTries--;
+                if (limitedTries == 0) {
+                    break;
+                }
+            }
+
+            System.out.println("-=-=-=-=-=-=-=-=-=-=-=-=-=-");
+            System.out.print("Word:   " + word + "\n");
+            System.out.print("Misses: " + missesChar + "\n");
+            System.out.print("Guess:  " + guessChar + "\n");
+            System.out.println("-=-=-=-=-=-=-=-=-=-=-=-=-=-");
+        }
+        if (word.toString().equals(chosenRandomWord)) {
+            System.out.println("YOU GOT IT!");
+        } else {
+            System.out.println("YOU LOST!");
+        }
+        playAgain();
```

Comment:
pašlaik sanāk ka `hangmanGame()` var izsaukt `playAgain()` un `playAgain()` izsauc `hangmanGame()`, thats how you can make infinite loops :smile:, labak šo ļoģiku ielikt kaut vai main, ja divas metodes var izsaukt viena otru, it can get ugly quite fast :smile: